filter_plugins/oo_filters.py

```diff
@@ -985,6 +997,18 @@ def oo_persistent_volume_claims(hostvars, persistent_volume_claims=None):
         persistent_volume_claim = oo_component_pv_claims(hostvars, 'metrics')
         if persistent_volume_claim is not None:
             persistent_volume_claims.append(persistent_volume_claim)
+    if 'prometheus' in hostvars['openshift']:
```

Comment:
@sdodson this sort of duplication in core modules is exactly what my underlying concern is with the coupling between our different components.  Having to do this is a code smell and is coupling completely unrelated to playbooks.